src/main/kotlin/com/salesforce/zero/quickquartz/QuickQuartz.kt

```diff
@@ -337,7 +337,7 @@ class QuickQuartz : JobStore, TablePrefixAware {
      */
     override fun storeJob(newJob: JobDetail?, replaceExisting: Boolean) {
         val job = newJob ?: throw IllegalArgumentException("job was null")
-        storeJobsAndTriggers(mutableMapOf(job to mutableSetOf()), replace = false)
+        storeJobsAndTriggers(mutableMapOf(job to mutableSetOf()), replaceExisting = false)
```

Comment:
Why = false?

Reply:
because replace=true requires implementing upsert, which is not there yet.

Reply:
my bad, fixing.